programs/solana/programs/ics27-gmp/tests/gmp_tests.rs

```diff
@@ -179,24 +179,6 @@ fn test_send_call_msg_validation_success() {
     );
 }
 
-#[test]
-fn test_send_call_msg_validation_timeout_too_soon() {
-    let current_time = 1_000_000;
-    let msg = SendCallMsg {
-        source_client: "cosmoshub-1".to_string(),
-        timeout_timestamp: current_time + 30, // Less than MIN_TIMEOUT_DURATION
-        receiver: Pubkey::new_unique(),
-        salt: vec![],
-        payload: vec![1, 2, 3],
-        memo: String::new(),
-    };
-
-    assert!(
-        msg.validate(current_time).is_err(),
-        "Timeout too soon should fail"
-    );
-}
```

Comment:
You don't need to remove this test even if MIN_TIMEOUT_DURATION is zero. You can test with something less than current_time.